### PR TITLE
converter python binary: revise input/output format option names

### PR DESCRIPTION
**Pull request by caisq**
- `--output_format`:
  - tensorflowjs --> tfjs_layers_model: for keras and keras_saved_model
    input formats
  - tensorflowjs --> tfjs_graph_model: for tf_frozen_model,  tf_hub,
    tf_saved_model, and tf_session_bundle input formats
- `--input_format`:
  - tensorflowjs --> tfjs_layers_model

BREAKING

Towards: https://github.com/tensorflow/tfjs/issues/1175
Fixes: https://github.com/tensorflow/tfjs/issues/1284

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/314)

### CODE REVIEW

**Comment by caisq** [comment]
@nsthorat I was thinking about the same question.

We have two options:
1) In the master branch, we'll not do the warning message. But we can create a separate branch, say 0.9.x, and release a 0.9.0 version of the pip package from it. There we can add some friendly error messages. This is similar to what we do for the JS packages.

2) Do the warning messages in the master branch.

I slightly prefer option 1, because it is consistent with what we do for the JS package. Let me know

@dsmilkov @pyu10055

**Comment by nsthorat** [comment]
I agree in principle with going for 1), but given that we don't have much time left and the work required to do that / manage another release branch, I think we should just keep it at master (especially since it's just a script, not in people's code).

We don't have to make it a warning, we can make it an error message so people actually update (and just say switch your string for the new one).

**Comment by dsmilkov** [comment]
+1 to just keep a single branch. Throw an error for using deprecated flags, pointing to the new flags.

File: python/tensorflowjs/converters/converter.py [hunk 1]
```diff
@@ -171,6 +171,58 @@ def dispatch_tensorflowjs_to_keras_h5_conversion(config_json_path, h5_path):
     model.save(h5_path)
     print('Saved Keras model to HDF5 file: %s' % h5_path)
 
+
+def _standardize_input_output_foramts(input_format, output_format):
```
Comment (pyu10055) [inline python/tensorflowjs/converters/converter.py]: _standardize_input_output_formats 
format mis spelled.

**Pull request merged**